pandas/core/series.py

```diff
@@ -3104,36 +3104,38 @@ def apply(self, func, convert_dtype=True, args=(), **kwds):
         """
         Invoke function on values of Series. Can be ufunc (a NumPy function
         that applies to the entire Series) or a Python function that only works
-        on single values
+        on single values.
```

Comment:
Can you have a single line description first, and the rest later. You have the documentation about this here:
https://pandas.pydata.org/pandas-docs/stable/contributing_docstring.html

If you run `./scripts/validate_docstrings.py pandas.Series.apply` should report an error about it.